benchexec/tablegenerator/react-table/src/workers/dataUrls.js

```diff
@@ -0,0 +1,11 @@
+// This file is part of BenchExec, a framework for reliable benchmarking:
```

Comment:
This file is checked in, but is generated and should only be necessary during the build process. Can we simply avoid checking it in, like we do with for example `dependencies.json`?

This would also mean we do not need to care about a proper license header.

If it is not checked in, we should maybe move it to the `build/` directory.

Reply:
I agree, as the file is created on every build, it can be removed. 
I've implemented the automated creation of the license header anyways.